src/docs/development/tools/devtools/memory.md

```diff
@@ -117,12 +149,18 @@ state of the memory as the application is running.
 The quantities plotted on the y-axis are (from top to bottom):
 
 <dl markdown="1">
+<dt markdown="1">**Legend**</dt>
+<dd>All collected measurements regarding the memory.
+    Clicking a legend name hides or shows that data.</dd>
+<dt markdown="1">**X-axis timestamp**</dt>
+<dd>Time of the collected memory information
+    (capacity, used, external, RSS (resident set size),
+    and GC (garbage collection).</dd>
+<dt markdown="1">**Hover information of collected data**</dt>
+<dd>At a particular time (x-axis) the detailed collected
```

Comment:
"The collected memory data at a particular point in time (x-axis)."

Reply:
Done.